gallery/src/pageConfig.ts

```diff
@@ -140,141 +140,154 @@ import NxStatefulCollapsibleRadioSelectPage
+    'Collapsible Items': { content: NxCollapsibleItemsPage, type: 'react' },
+    List: { content: NxListPageV2, type: 'react' },
+    Modal: { content: NxModalPage, type: 'react' },
+    Pagination: { content: NxPaginationPage, type: 'react' },
+    'Indeterminate Pagination': { content: NxIndeterminatePaginationPage, type: 'react' },
+    'Search Dropdown': { content: NxSearchDropdownPage, type: 'react' },
+    'Stateful Search Dropdown': { content: NxStatefulSearchDropdownPage, type: 'react' },
+    'Submit Mask': { content: NxSubmitMaskPage, type: 'react' },
+    'Stateful Submit Mask': { content: NxStatefulSubmitMaskPage, type: 'react' },
+    'System Notice': { content: NxSystemNoticePage, type: 'html' },
+    Table: { content: NxTablePage, type: 'react' },
+    'Table Container': { content: NxTableContainerPage, type: 'html' },
+    Tabs: { content: NxTabsPage, type: 'react' },
+    'Stateful Tabs': { content: NxStatefulTabsPage, type: 'react' },
+    Tooltip: { content: NxTooltipPage, type: 'react' },
+    'Overflow Tooltip': { content: NxOverflowTooltipPage, type: 'react' },
+    Tree: { content: NxTreePage, type: 'react' },
+    'Tree View': { content: NxTreeViewPage, type: 'react' },
+    'Tree View Multi-Select': { content: NxTreeViewMultiSelect, type: 'react' },
+    'Stateful Tree View Multi-Select': { content: NxStatefulTreeViewMultiSelect, type: 'react' },
+    'Tree View Radio Select': { content: NxTreeViewRadioSelectPage, type: 'react' },
+    'Stateful Tree View Radio Select': { content: NxStatefulTreeViewRadioSelectPage, type: 'react' }
   },
-  'Styles - Mixins & Helpers': {
-    'Color Palettes': ColorPalettePage,
-    'Custom app font size': NxFontSizePage,
-    'Ellipsis Truncation': NxTruncatePage,
-    'nx-clickable': NxClickablePage,
-    'nx-container-helpers': NxContainerHelpersPage,
-    'nx-scrollable': NxScrollablePage,
-    'nx-viewport-sized': NxViewportSizedPage,
-    'Selectable Colors': SelectableColorsPage,
-    'Style Variables': CssVariablesPage
+  'Typography': {
+    'Blockquote': { content: NxBlockquotePage, type: 'html' },
+    'Code': { content: NxCodePage, type: 'html' },
+    'H*': { content: NxHPage, type: 'html' },
+    'P': { content: NxPPage, type: 'html' },
+    'Pre': { content: NxPrePage, type: 'html' }
   },
-  'Layout Examples': {
-    'Form Layout Styles': NxFormLayoutPage,
-    'Page Layout': PageLayoutPage,
-    'Read-Only Grid Tile Layout': NxReadOnlyGridTilePage
+  'HTML Variants': {
+    'Alert': { content: NxAlertPage, type: 'html' },
+    'Button': { content: NxBtnPage, type: 'html' },
+    'Icon': { content: NxIconPage, type: 'html' },
+    'Fieldset': { content: NxFieldsetStylePage, type: 'html' },
+    'Form Group': { content: NxFormGroupStylePage, type: 'html' },
+    'Form Select': { content: NxFormSelectPage, type: 'html' },
+    'List': { content: NxListPage, type: 'html' },
+    'Table': { content: NxTableStylePage, type: 'html' }
+  },
+  'Mixins & Helpers': {
```

Comment:
Many of these have nothing to do with Sass in particular, and are just CSS variables and classes.  For those ones, I think a different logo should be used that isn't the Sass logo

Reply:
> Many of these have nothing to do with Sass in particular, and are just CSS variables and classes. For those ones, I think a different logo should be used that isn't the Sass logo

The problem is that the CSS icons are terrible. I'll see what I can do.

Reply:
> > Many of these have nothing to do with Sass in particular, and are just CSS variables and classes. For those ones, I think a different logo should be used that isn't the Sass logo
> 
> The problem is that the CSS icons are terrible. I'll see what I can do.

So ugly but added: https://github.com/sonatype/sonatype-react-shared-components/pull/596/commits/950ffa612130f2a1927326c712ef2e689aad5a1c

Reply:
> So ugly but added: 950ffa6

I wonder if we could take the "CSS" part of this: https://commons.wikimedia.org/wiki/File:CSS3_and_HTML5_logos_and_wordmarks.svg

Pretty silly that the only part that made it into the FA icon is the "3".  Especially since half of CSS is on 4 now.

Reply:
> > So ugly but added: [950ffa6](https://github.com/sonatype/sonatype-react-shared-components/commit/950ffa612130f2a1927326c712ef2e689aad5a1c)
> 
> I wonder if we could take the "CSS" part of this: https://commons.wikimedia.org/wiki/File:CSS3_and_HTML5_logos_and_wordmarks.svg
> 
> Pretty silly that the only part that made it into the FA icon is the "3". Especially since half of CSS is on 4 now.

Added an svg version here: 05bb000385b339d31e3c8702747596709fb0aa3e
I adjusted the mark to work on that scale.

Reply:
Fixed it here: 7a994245f0c4f024ff013827fb7c09096d4bc46c